src/common/components/repository-row/index.js

```diff
@@ -1,37 +1,75 @@
-import React, { PropTypes } from 'react';
+import React, { Component, PropTypes } from 'react';
 import { Link } from 'react-router';
 
-import { Row, Data } from '../vanilla/table-interactive';
+import { Row, Data, Dropdown } from '../vanilla/table-interactive';
 import BuildStatus from '../build-status';
 
 import { parseGitHubRepoUrl } from '../../helpers/github-url';
 
-const RepositoryRow = (props) => {
-  const { snap, latestBuild } = props;
-  const { fullName } = parseGitHubRepoUrl(snap.git_repository_url);
-
-  return (
-    <Row>
-      <Data col="30"><Link to={ `/${fullName}/builds` }>{ fullName }</Link></Data>
-      <Data col="20"> </Data>
-      <Data col="20"> </Data>
-      <Data col="30">
-        {/*
-          TODO: show 'Loading' when waiting for status?
-            and also show 'Never built' when no builds available
-        */}
-        { latestBuild &&
-          <BuildStatus
-            link={ `/${fullName}/builds/${latestBuild.buildId}`}
-            status={ latestBuild.status }
-            statusMessage={ latestBuild.statusMessage }
-            dateStarted={ latestBuild.dateStarted }
-          />
-        }
-      </Data>
-    </Row>
-  );
-};
+class RepositoryRow extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      notConfiguredDropdownExpanded: false
```

Comment:
True. It came from 'Not configured' snap state, but I agree that in the longer variable name it's confusing what 'not' is negating actually ;)